src/ray/gcs/gcs_client.h

```diff
@@ -7,12 +7,18 @@
 #include <vector>
 #include "ray/common/status.h"
 #include "ray/gcs/actor_info_accessor.h"
+#include "ray/gcs/job_info_accessor.h"
 #include "ray/util/logging.h"
 
 namespace ray {
 
 namespace gcs {
 
+/// Specifies whether commands issued to a table should be regular or chain-replicated
+/// (when available).
+// TODO(micafan) Rename to RedisCommandType and maybe move it to gcs/type.h
+enum class CommandType { kRegular, kChain, kUnknown };
```

Comment:
this is specific to redis gcs client and should be moved to that file?

Reply:
Sorry, I made a mistake.